tests/rest/client/test_relations.py

```diff
@@ -93,11 +93,6 @@ def test_send_relation(self):
             channel.json_body,
         )
 
-    def test_deny_membership(self):
-        """Test that we deny relations on membership events"""
-        channel = self._send_relation(RelationTypes.ANNOTATION, EventTypes.Member)
-        self.assertEquals(400, channel.code, channel.json_body)
```

Comment:
As far as I can tell there's not need to test this specifically?

Reply:
Not quite sure how to confirm that, but the linked MSC didn't have the word "member" or "membership" in it anywhere.

Reply:
I'm not quite sure either. MSC2677 also doesn't have any mention...

Anyway, I removed this because the client-server API won't let you make a membership event directly anyway, so I don't see why this is an issue!